test/testvfdswmr.sh.in

```diff
@@ -1083,6 +1083,16 @@ for options in "-d 1" "-d 1 -F" "-d 2 -l 16" "-d 2 -F -l 16" "-d 1 -t" "-d 1 -t
         # first dimension (up to 25 1x16x16)
         #
 
+        # Launch the auxiliary process for testing NFS if the --enable-aux-process option is enabled for configuration.
```

Comment:
Minor: is the indentation for line #1086 to #1095 intentional? or should they be lined up with the rest of the code?

Reply:
Intentional.